src/current/_includes/v23.2/sql/grant-revoke-schema-changes.md

```diff
@@ -1,9 +1,9 @@
 
-For example, schema changes wait for concurrent transactions using the same resources as the schema changes to complete. In the case of [role memberships]({% link {{ page.version.version }}/security-reference/authorization.md %}#roles) being modified by `GRANT` and `REVOKE` inside a transaction, most transactions need access to the set of role memberships.
+For example, schema changes wait for concurrent [transactions]({% link {{ page.version.version }}/transactions.md %}) using the same resources as the schema changes to complete. In the case of [role memberships]({% link {{ page.version.version }}/security-reference/authorization.md %}#roles) being modified inside a transaction, most transactions need access to the set of role memberships. Using the default settings, any modifications to roles require schema leases to expire, which can take up to 5 minutes.
 
-This means that [long-running transactions]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#hanging-or-stuck-queries) elsewhere in the system can cause `GRANT` and `REVOKE` operations inside transactions to take several minutes to complete. This can have a cascading effect. When a `GRANT` or `REVOKE` operation inside a transaction takes a long time to complete, it can in turn block all user-initiated transactions being run by your application, since the `GRANT` operation in the transaction has to commit before any other transactions that access role memberships (i.e., most transactions) can make progress.
+This means that [long-running transactions]({% link {{ page.version.version }}/query-behavior-troubleshooting.md %}#hanging-or-stuck-queries) elsewhere in the system can cause user/role management operations inside transactions to take several minutes to complete. This can have a cascading effect. When a user/role management operation inside a transaction takes a long time to complete, it can in turn block all user-initiated transactions being run by your application, since the user/role management operation in the transaction has to commit before any other transactions that access role memberships (i.e., most transactions) can make progress.
 
-If you would prefer that grant and revoke operations finish rapidly, and do not care whether concurrent transactions will immediately see the side-effects of those operations, set the session variable `allow_role_memberships_to_change_during_transaction` to `true`.
+If you want user/role management operations to finish more quickly, and do not care whether concurrent transactions will immediately see the side-effects of those operations, set the [session variable]({% link {{ page.version.version }}/set-vars.md %}) `allow_role_memberships_to_change_during_transaction` to `true`.
```

Comment:
Lets some how get an example to globally apply this to a cluster:
ALTER ROLE ALL SET allow_role_memberships_to_change_during_transaction = true;

Reply:
Good idea! I revised the end of this a bit to say (basically) you can either pass it on the conn string OR you can do ALTER ROLE ALL - PTAL and let me know what you think!